code/game/gamemodes/wizard/apprentice_contract.dm

```diff
@@ -161,7 +161,7 @@ var/list/wizard_apprentice_setups_by_name = list()
 	to_chat(player, "<span class='recruit'>\A [src] is looking for candidates. You have been added to the list of potential ghosts. ([controls])</span>")
 
 /obj/item/wizard_apprentice_contract/proc/recruiter_not_recruiting(mob/dead/observer/player, controls)
-	var/mob/dead/observer/O = args["player"]
+	var/mob/dead/observer/O = player
```

Comment:
it's already the right type, no need to cast

Reply:
right